care/facility/models/asset.py

```diff
@@ -58,7 +58,7 @@ class Status(enum.Enum):
     not_working_reason = models.CharField(max_length=1024, blank=True, null=True)
     serial_number = models.CharField(max_length=1024, blank=True, null=True)
     warranty_details = models.TextField(null=True, blank=True, default="")
-    meta = JSONField(default=dict)
+    meta = JSONField(null=True, blank=True)
```

Comment:
To make the field optional making it nullable is not required
```suggestion
    meta = JSONField(default=dict, blank=True)
```